k-distribution/k-tutorial/1_basic/02_basics/README.md

```diff
@@ -58,41 +76,81 @@ We can now evaluate this K term by running (from the same directory):
 krun banana.color
 ```
 
-`krun` will use the interpreter generated by the first call to `kompile` to
-execute this program.
+`krun` is a tool which uses the interpreter generated by `kompile` to execute 
+this program.
 
-You will get the following output:
+You should get the following output:
 
 ```
 <k>
-  Yellow ( ) ~> .
+  Yellow ( ) ~> .K
 </k>
 ```
 
-For now, don't worry about the `<k>`, `</k>`, or ` ~> .` portions of this
-output file.
+For now, don't worry about the `<k>`, `</k>`, or ` ~> .K` portions of this
+output file. We'll come back to them later.
 
-You can also execute small programs directly by specifying them on the command
-line instead of putting them in a file. For example, the same program above
+You can also execute small programs directly on the command line instead of 
+putting them in a file. For example, the same program above
 could also have been executed by running the following command:
 
 ```
 krun -cPGM='colorOf(Banana())'
 ```
 
-Now, let's look at what this definition and program did.
+You could also run it as:
+
+```
+krun -cPGM='colorOf(Banana())' --definition 'lesson-02-a-kompiled'
+```
+
+Attribute `--definition` points to the *directory* containing the compiled version
+of `LESSON-02-A`.
+
+### Exercise
+
+Use `krun` to compute the return value of the `colorOf` function on a
+`Blueberry()`.
+
+## Structure of a K file
+
+A K file consists of one or more **modules**, of which only one is the 
+_main_ one, in the sense that no other module defined in that file **imports** 
+it. The name of this _main_ module will also be the one given as argument to 
+attribute `main-module` at compile time. 
+Modules residing in other K files are imported instead through **requires** 
+statements (similar to `include` in C/C++). You'll learn more about **imports** and 
+**requires** in [Lesson 1.5](../05_modules/README.md).
+In this lesson we continue with a closer look to a K module syntax.
```

Comment:
I would avoid using “syntax” here if possible. We already have the keyword `syntax` and the syntax module. So, a synonym for “structure” would be better in this case.